dataflow/gemma-flex-template/README.md

```diff
@@ -0,0 +1,177 @@
+# RunInference on Dataflow streaming with Gemma and Flex Templates
+
+Gemma is a family of lightweight, state-of-the-art open models built from research and technology used to create the Gemini models.
+You can use Gemma models in your Apache Beam inference pipelines with the `RunInference` transform.
+
+This example demonstrates how to use a Gemma model running on Pytorch in a streaming Dataflow pipeline that has Pub/Sub sources and sinks. This pipeline
+is deployed by using Flex Templates.
+
+For more information about using RunInference, see [Get started with AI/ML pipelines](https://beam.apache.org/documentation/ml/overview/) in the Apache Beam documentation.
+
+## Before you begin
+
+Make sure you have followed the [Dataflow setup instructions](../../README.md).
```

Comment:
This relative path points to the main repo README, not the dataflow README. Suggest using absolute path
```suggestion
Make sure you have followed the [Dataflow setup instructions](/dataflow/README.md).
```